fe/fe-core/src/main/java/com/starrocks/external/starrocks/TableMetaSyncer.java

```diff
@@ -29,7 +29,7 @@ public void syncTable(ExternalOlapTable table) {
         authInfo.setPasswd(table.getSourceTablePassword());
         request.setAuth_info(authInfo);
         try {
-            TGetTableMetaResponse response = FrontendServiceProxy.call(addr, 10000,
+            TGetTableMetaResponse response = FrontendServiceProxy.call(addr, 10000, 3,
                     client -> client.getTableMeta(request));
```

Comment:
1. Give the description of the bug and the solution in the commit message.
2. Could the retry can solve the multi-followers' problem about the configuration?

Reply:
1. added.
2. yes, the reason of this bug #5655 is that frontendPool returned a broken conn.

Reply:
Under what circumstance, will the broker connection be returned from the connection pool?
Could the broker connection keep-alive to avoid the broken connection?

Reply:
I will continue to look for this reason, and make another pr to fix the connection poll bug. But retry is a must when fe restarted or network problem occurred.